src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -130,6 +130,7 @@ internal enum MessageID
 
         IDS_FeatureReadonlyReferences = MessageBase + 12718,
 
+        IDS_SK_CONSTRUCTOR = MessageBase + 12719,
```

Comment:
Group with other `IDS_SK_` values in the file and use value in the same range, say `= MessageBase + 2013`.